hr_expense_sequence/report/report_expense_sheet.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why this is removed?

Reply:
@alexey-pelykh Sorry, I misunderstand. Edied.